package.json

```diff
@@ -10,7 +10,7 @@
     "build": "tsc",
     "watch": "tsc -w",
     "extensions:install": "func extensions install",
-    "start": "dotenv -e .env func start --javascript",
+    "start": "dotenv -e .env.prod func start --javascript",
```

Comment:
Why `.env.prod` instead of `.env` for local development?

Reply:
`.env` is used by `docker-compose` running, `.env.prod` is used to run the backend with production/stage configuration for database and Azure storage services

Reply:
We could use anyway the same `.env` file, changing the values if you think that is more clear.

Reply:
Fixen in 026bdb2